doc/release-notes/0.49/0.49.md

```diff
@@ -0,0 +1,145 @@
+
+<table cellpadding="4" cellspacing="0" summary="" width="100%" rules="all" frame="border" border="1"><thead align="left">
+<tr>
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Version / Platform</th>
+<th valign="bottom">Impact</th>
+</tr>
+</thead>
+<tbody>
+
+<tr>
+<td valign="top">N/A</td>
+<td valign="top">macOS&reg; 12 is out of support</td>
+<td valign="top">All versions (macOS 12)</td>
+<td valign="top">macOS 13 is the new minimum operating system level.</td>
+</tr>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/20029">#20029</a></td>
+<td valign="top">The shared classes cache generation number is incremented.</td>
+<td valign="top">All versions</td>
+<td valign="top">The increment in the shared classes cache generation number causes the VM to create a new shared classes cache, rather than re-creating or reusing an existing cache.
+The shared classes cache generation number is modified because of a change in the format of ROMClasses that are stored in the shared classes cache. A new flag <tt>J9AccClassIsShared</tt> is added to ROMClasses to indicate whether a ROMClass was loaded from a shared classes cache or from a VM.
+</td>
+</tr>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/pull/20366">#20366</a></td>
+<td valign="top">A new shared classes cache suboption <tt>-Xshareclasses:extraStartupHints</tt> is added.</td>
+<td valign="top">All versions</td>
+<td valign="top">By default, you can add up to 64 startup hints only in a shared cache. You can use the <tt>-Xshareclasses:extraStartupHints=&ltnumber&gt</tt> option to adjust the number of startup hints that can be stored in a shared cache in addition to the default number.</td>
+</tr>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/pull/19872">#19872</a></td>
+<td valign="top"><tt>subAllocator</tt> related <tt>-Xgc</tt> options are added to control the compressed reference allocation.</td>
+<td valign="top">All versions</td>
+<td valign="top">The VM can use compressed references to decrease the size of Java objects and make better use of the available space in the Java heap. You can now control the compressed reference allocation with the following options:
+
+ - <tt>-Xgc:suballocatorCommitSize=\<size></tt>
+ - <tt>-Xgc:suballocatorIncrementSize=\<size></tt>
+ - <tt>-Xgc:suballocatorInitialSize=\<size></tt>
+ - <tt>-Xgc:suballocatorQuickAllocDisable</tt>
+ - <tt>-Xgc:suballocatorQuickAllocEnable</tt></td>
+</tr>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/pull/20854">#20854</a></td>
+<td valign="top">Support for the Java Flight Recorder (JFR) tool in the VM is provided as a technical preview and a new option <tt>-XX:[+|-]FlightRecorder</tt> is added to enable or disable the JFR tool.</td>
```

Comment:
I think the word "tool" should be removed from four places. It diagnostic output from the JVM, not a tool.
- Support for Java Flight Recorder (JFR) in the VM
- to enable or disable JFR
- JFR is enabled by default
- You can disable JFR with

Reply:
Btw, there is a `bin/jfr` tool that read the .jfr file and prints information, but that's not what we are talking about here.